pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1786,6 +1803,61 @@ func (c *Cloud) DisksAreAttached(diskNames []KubernetesVolumeID, nodeName types.
 	return attached, nil
 }
```

Comment:
I am half thinking that may be - we should delete the whole `DisksAreAttached` and `VolumesAreAttached` functions and make `VolumesAreAttached` function for aws log a giant warning/error when it gets used here. Because really no one should be using it.